.gitignore

```diff
@@ -22,9 +23,9 @@ tags
 /docs/build
 /ployst.egg-info
 /static
+/.tox/
 
 # Some OSX specific thing
 *.DS_Store
-/.tox/
 
 ployst/github/test/data/
```

Comment:
Might be because we clone the pretenders/dummy repo there in some tests.

Perhaps the path should be extended to ``ployst/github/test/data/pretenders``